### PR TITLE
Blank slate for no portfolios

### PR DESCRIPTION
**Pull request by drummond-dds**
New user landing page for when a user has no portfolios. We can ship it without the "sad icon" in the ticket, and the `TODO` text will be added in a follow up PR.

[Pivotal Tracker](https://www.pivotaltracker.com/story/show/163272393).

screen shot 2019-02-11 at 16 07 38

### CODE REVIEW

File: templates/portfolios/blank_slate.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      icon="cloud",
+      message=("portfolios.index.empty.title" | translate),
+    )
+  }}
+
+  <div class="center">
+    {{
+      Tooltip(
+        message=("portfolios.index.empty.help_message" | translate),
+        title=("portfolios.index.empty.help_title" | translate),
```
Comment (patricksmithdds) [inline templates/portfolios/blank_slate.html]: I'm curious if it's better to either leave this out entirely if we just have a "todo" message, or come up with a message ourselves?

Comment (drummond-dds) [reply templates/portfolios/blank_slate.html]: Good idea, removed it in f22fda4e28d3833b9e4313011418d4801b1bb217

Comment (patricksmithdds) [reply templates/portfolios/blank_slate.html]: Should we remove the _whole_ tooltip? I don't think there's a reason to have it in there unless we can actually give the user some contextual info.

Comment (drummond-dds) [reply templates/portfolios/blank_slate.html]: Makes sense - I removed that in 6172133

**Pull request merged**